hoomd/md/pytest/test_pair.py

```diff
@@ -0,0 +1,939 @@
+                         [0.103803, -0.000320337],
+                         [24.192, 0]]
+    forces["DPDLJ"] = [[2.11492, 1.20788],
+                       [64.2114, 37.7452],
+                       [-377.586, 7.15179]]
+
+    params["DLVO"] = [{'kappa': 1.0, 'Z': 0.1, 'A': 0.1},
+                      {'kappa': 2.0, 'Z': 0.5, 'A': 0.5},
+                      {'kappa': 5.0, 'Z': 2.0, 'A': 2.0}]
+    energies["DLVO"] = [[0.00476409, 0.00226142],
+                        [0.0159279, 0.00357933],
+                        [0.0188554, 0.000449299]]
+    forces["DLVO"] = [[-0.00456658, -0.00226058],
+                      [-0.0309878, -0.00715578],
+                      [-0.0922397, -0.00225115]]
+
+    param_list = []
+    for pair_potential in params.keys():
+        kT_dict = {}
+        if pair_potential == "DPD":
+            kT_dict = {"kT": 2}
+        elif pair_potential == "DPDLJ":
+            kT_dict = {"kT": 1}
+        for i in range(3):
+            param_list.append((pair_potential,
+                               getattr(hoomd.md.pair, pair_potential),
+                               kT_dict,
+                               params[pair_potential][i],
+                               forces[pair_potential][i],
+                               energies[pair_potential][i]))
+    return param_list
+
+
+@pytest.fixture(scope="function",
+                params=_forces_and_energies(),
+                ids=(lambda x: x[0]))
+def forces_and_energies(request):
+    return deepcopy(request.param)
+
+
+def _assert_equivalent_type_params(type_param1, type_param2):
+    for pair in type_param1:
+        if isinstance(type_param1[pair], dict):
+            for key in type_param1[pair]:
+                np.testing.assert_allclose(type_param1[pair][key],
+                                           type_param2[pair][key])
+        else:
+            assert type_param1[pair] == type_param2[pair]
+
+
+def _assert_equivalent_parameter_dicts(param_dict1, param_dict2):
+    for key in param_dict1:
+        assert param_dict1[key] == param_dict2[key]
+
+
+def test_rcut(simulation_factory, two_particle_snapshot_factory):
+    lj = hoomd.md.pair.LJ(nlist=hoomd.md.nlist.Cell(), r_cut=2.5)
+    lj.params[('A', 'A')] = {'sigma': 1, 'epsilon': 0.5}
+    with pytest.raises(hoomd.typeconverter.TypeConversionError):
+        lj.r_cut[('A', 'A')] = 'str'
+    with pytest.raises(hoomd.typeconverter.TypeConversionError):
+        lj.r_cut[('A', 'A')] = [1, 2, 3]
+
+    sim = simulation_factory(two_particle_snapshot_factory(dimensions=3, d=.5))
+    integrator = hoomd.md.Integrator(dt=0.005)
+    integrator.forces.append(lj)
+    integrator.methods.append(hoomd.md.methods.Langevin(hoomd.filter.All(),
+                                                        kT=1, seed=1))
+    sim.operations.integrator = integrator
+
+    _assert_equivalent_type_params(lj.r_cut.to_dict(), {('A', 'A'): 2.5})
+    sim.operations.schedule()
+    sim.run(1)
```

Comment:
```suggestion
    sim.run(1)
```
We don't need to explicitly call schedule any more.